tests/squash/squash.sh

```diff
@@ -8,7 +8,7 @@ if grep -q "Red Hat Enterprise Linux release 8" /etc/system-release; then
   exit 0
 fi
 
-origin=busybox
+origin=quay.io/centos7/s2i-core-centos7
```

Comment:
Do we need to use s2i-core here? I would use something small if we do not need any of s2i-core's functionality

Reply:
ok, It should be from quay.io or registry.redhat.io. Not from docker hub. May by ubi7/ubi.

I will test it.

Reply:
@pkubatrh registry.access.redhat.com/ubi8-micro does not make sense, I think. For RHEL8, squash is ignored https://github.com/sclorg/container-common-scripts/blob/master/tests/squash/squash.sh#L7.

But what about using registry.access.redhat.com/ubi7/ubi? ubi8-micro is not released yet;)